### PR TITLE
Simplify cube region generation code.

### PR DESCRIPTION
**Pull request by Hoikas**
This consolidates a very common bit of LogicWiz modifier boilerplate into one location for ease of use.

### CODE REVIEW

**Comment by Hoikas** [comment]
Lack of feedback interpreted as implicit approval.

File: korman/operators/op_mesh.py [hunk 1]
```diff
@@ -100,15 +100,15 @@ def create_flare_objects(self):
...
         # Create a textured Plane
-        with utils.bmesh_object("{}_Visible".format(self.name_stem)) as (flare_plane, bm):
-            flare_plane.hide_render = True
-            flare_plane.plasma_object.enabled = True
-            bpyscene.objects.active = flare_plane
-
+        flare_plane = utils.BMeshObject(f"{self.name_stem}_Visible", managed=False)
+        flare_plane.hide_render = True
+        flare_plane.plasma_object.enabled = True
+        bpyscene.objects.active = flare_plane
```
Comment (Deledrius) [inline korman/operators/op_mesh.py:106]: > TypeError: BPy_PropertyRNA - Attribute (setattr): SceneObjects.active expected a Object type, not BMeshObject

**Review by Deledrius — COMMENTED** [review]
This PR appears to be causing an error when I attempt to create a flare object.

File: korman/operators/op_mesh.py (hunk 1 above)
Comment (Deledrius) [reply korman/operators/op_mesh.py:106]:
```suggestion
        bpyscene.objects.active = flare_plane.object
```

Comment (Deledrius) [reply korman/operators/op_mesh.py:106]: This suggestion fixes the export error, but the flare is still not visible.

File: korman/operators/op_mesh.py [hunk 2]
```diff
@@ -100,15 +100,15 @@ def create_flare_objects(self):
         flare_root.plasma_modifiers.viewfacemod.preset_options = "Sprite"
 
         # Create a textured Plane
-        with utils.bmesh_object("{}_Visible".format(self.name_stem)) as (flare_plane, bm):
-            flare_plane.hide_render = True
-            flare_plane.plasma_object.enabled = True
-            bpyscene.objects.active = flare_plane
-
+        flare_plane = utils.BMeshObject(f"{self.name_stem}_Visible", managed=False)
+        flare_plane.hide_render = True
```
Comment (Deledrius) [inline korman/operators/op_mesh.py:104]: Missing object property.

Comment (Deledrius) [reply korman/operators/op_mesh.py:104]:
```suggestion
        flare_plane.object.hide_render = True
```

File: korman/operators/op_mesh.py [hunk 3]
```diff
@@ -100,15 +100,15 @@ def create_flare_objects(self):
...
+        flare_plane.plasma_object.enabled = True
```
Comment (Deledrius) [inline korman/operators/op_mesh.py:105]: Missing object property.

Comment (Deledrius) [reply korman/operators/op_mesh.py:105]:
```suggestion
        flare_plane.object.plasma_object.enabled = True
```

File: korman/operators/op_mesh.py (hunk 1 above)
Comment (Deledrius) [reply korman/operators/op_mesh.py:106]: L#120 & L#123 also require flare_plane.object.  Alas, I cannot leave suggestions on unedited lines.

**Comment by Deledrius** [comment]
> Lack of feedback interpreted as implicit approval.

Interpretation incorrect.  Feedback submitted.  Changes required.  ;)

**Pull request merged**